src/components/Reown/SendTransactionModal.js

```diff
@@ -0,0 +1,439 @@
+    fontSize: 20,
+    fontWeight: 'bold',
+    textAlign: 'center',
+  },
+  loadingContainer: {
+    alignItems: 'center',
+    justifyContent: 'center',
+    padding: 24,
+  },
+  loadingText: {
+    fontSize: 16,
+    fontWeight: 'bold',
+    marginTop: 16,
+    textAlign: 'center',
+  },
+  statusContainer: {
+    alignItems: 'center',
+    justifyContent: 'center',
+    padding: 24,
+  },
+  statusText: {
+    fontSize: 16,
+    fontWeight: 'bold',
+    marginTop: 16,
+    textAlign: 'center',
+  },
+  statusMessage: {
+    fontSize: 14,
+    marginTop: 8,
+    textAlign: 'center',
+    color: 'hsla(0, 0%, 38%, 1)',
+  },
+  successColor: {
+    color: 'green',
+  },
+  errorColor: {
+    color: 'red',
+  },
+});
+
+export default ({
+  onDismiss,
+  data,
+  onAcceptAction,
+  onRejectAction,
+}) => {
+  const navigation = useNavigation();
+  const dispatch = useDispatch();
+  const { tokens: registeredTokens } = useSelector((state) => ({
+    tokens: state.tokens,
+  }));
+
+  const getTokenSymbol = (tokenId) => {
+    if (!tokenId) {
+      return constants.DEFAULT_NATIVE_TOKEN_CONFIG.symbol;
+    }
+
+    const token = registeredTokens[tokenId];
+    if (token) {
+      return token.symbol;
+    }
+
+    return '?';
+  };
+
+  const formatValue = (value) => {
+    if (value == null) {
+      return '-';
+    }
+
+    return numberUtils.prettyValue(value);
+  };
+
+  const truncateTxId = (txId) => {
+    if (!txId) return '-';
+    return `${txId.slice(0, 12)}...${txId.slice(-12)}`;
+  };
+
+  const copyToClipboard = (text) => {
+    Clipboard.setString(text);
+  };
+
+  const renderInputs = () => {
+    if (!data?.data?.inputs || data.data.inputs.length === 0) {
+      return null;
+    }
+
+    return (
+      <View>
+        <Text style={styles.sectionTitle}>{t`Inputs`}</Text>
+        {data.data.inputs.map((input, index) => {
+          const inputKey = `input-${input.txId || ''}-${input.index || ''}-${index}`;
+          return (
+            <View
+              key={inputKey}
+              style={styles.itemContainer}
+            >
+              <View style={styles.itemHeader}>
+                <Text style={styles.itemTitle}>{t`Input ${index + 1}`}</Text>
+                <Text style={styles.valueText}>
+                  {formatValue(input?.value)} {getTokenSymbol(input?.token)}
+                </Text>
+              </View>
+              <Text style={styles.labelText}>{t`ID da Transação`}</Text>
+              <View style={styles.valueContainer}>
+                <Text style={styles.monospace}>
+                  {truncateTxId(input?.txId)} ({input?.index})
+                </Text>
+                {input?.txId && (
+                  <TouchableOpacity onPress={() => copyToClipboard(input.txId)}>
+                    <Text style={styles.copyButton}>{t`Copy ID`}</Text>
+                  </TouchableOpacity>
+                )}
+              </View>
+              <Text style={styles.labelText}>{t`Endereço`}</Text>
```

Comment:
```suggestion
              <Text style={styles.labelText}>{t`Address`}</Text>
```

Reply:
Fixed, thanks

The fix is in SendTransactionRequest component